_config.yml

```diff
@@ -17,17 +17,14 @@ phone: (716) 555-5555
 tiles-source: pages # accepts "posts" or "pages"
 tiles-count: 6
 
-# social settings
-500px_url:
-facebook_url:
-github_url:
-gitlab_url: https://gitlab.com/andrewbanchich
-googleplus_url: https://plus.google.com/u/0/+AndrewBanchich
-instagram_url: https://instagram.com
-linkedin_url: https://www.linkedin.com/in/andrew-banchich-a4ba1195
-pinterest_url:
-slack_url:
-twitter_url: https://twitter.com/andrewbanchich
+# social settings (key must match name of font awesome icon)
+# see https://fontawesome.com/icons?d=gallery&p=2&s=brands
+socials:
```

Comment:
This looks great! Is it possible to include GitLab?

Reply:
I appended the commit to allow for empty keys in the list so that the sample config.yml can list other available social networks.